.changeset/legal-lines-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
+---
+
```

Comment:
I don't think we need a changeset as these changes do not affect core node release process